changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="utf-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Marcin Mogiela" id="1">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet author="Marcin Mogiela" id="2">
+    <createTable tableName="eligibility">
+      <column name="sbi" type="NUMBER(9)">
+        <constraints nullable="true" unique="true" uniqueConstraintName="elg_sbi_ukey" />
```

Comment:
I think we can remove the nullable part for all columns. Maybe @steve-dickinson  can confirm that one?

Reply:
done